src/qz/printer/PrintServiceMatcher.java

```diff
@@ -27,9 +27,13 @@
 import javax.print.attribute.standard.PrinterName;
 import javax.print.attribute.standard.PrinterResolution;
 import java.util.Locale;
+import java.util.concurrent.TimeUnit;
 
 public class PrintServiceMatcher {
     private static final Logger log = LogManager.getLogger(PrintServiceMatcher.class);
+    private static final long cachedDefaultServiceLifespan = TimeUnit.SECONDS.toNanos(10); //Set to 0 to disable, do not set to a negative number
```

Comment:
Why a default of 10 nanoseconds specifically?  What's the reasoning behind this value?

Reply:
That is actually 10 seconds, in nanoseconds. it could also be expressed as 10 
* 1000000000.